_posts/app/2018-02-07-metrics.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Applications Metrics
+modified_at: 2018-02-07 00:00:00
+category: app
+tags: app metrics
+---
+
+We gather a few performance metrics of your application that are accessible on the dashboard of your
```

Comment:
Not `We`, `Performance metrics from applications are gathered and are accessible from the dashboard`